tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_download_feeds.yaml

```diff
@@ -90,6 +90,18 @@
     provider_os: 'FOCAL'
     update_treshold_weeks: 2
 
+- name: 'JAMMY'
+  description: 'Canonical provider'
+  configuration_parameters:
+    PROVIDER: 'canonical'
+    OS: 'jammy'
+  metadata:
+    provider_name: 'Ubuntu Jammy'
+    download_timeout: 120
+    provider_os: 'JAMMY'
+    update_treshold_weeks: 2
```

Comment:
Did you check that it takes 2 weeks?

Reply:
Could not find any specific information on what it takes exactly.